core/sail/solr/pom.xml

```diff
@@ -14,6 +14,7 @@
 		<enforce-javaee-provided.fail>false</enforce-javaee-provided.fail>
 		<!-- use at least 3.7 to fix CVE -->
 		<zookeeper.version>3.7.2</zookeeper.version>
+		<snappy.version>1.1.10.5</snappy.version>
```

Comment:
Could you add a comment here similar to the one above so that we know that we are only including the version of snappy here to fix a CVE and that it’s not something that RDF4J is using directly?

Reply:
ok, done